src/Core/Managed/Shared/Extensibility/Implementation/SdkVersionUtils.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation
+{
+    using System;
+    using System.Globalization;
+    using System.Linq;
+    using System.Reflection;
+
+    internal class SdkVersionUtils
+    {
+        /// <summary>
+        /// Builds a string representing file version of the assembly with added prefix
+        /// in format prefix:major.minor-revision.
+        /// </summary>
+        /// <param name="versionPrefix">Prefix to add to version.</param>
+        /// <returns>String representation of the version with prefix added.</returns>
+        internal static string GetSdkVersion(string versionPrefix)
+        {
+#if !CORE_PCL
```

Comment:
This is code I *moved* from TelemetryClient class out so that it can be used in two places. I did not modify the code.